src/index.html

```diff
@@ -4,13 +4,34 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+              user-scalable=no,
+              initial-scale=1.0,
+              maximum-scale=1.0,
+              minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="block__input input-big" data-qa="big">
+      <div class="input-big__wrapp input-wrapp">
+        <input
+          class="input-big__item input"
```

Comment:
It will be awesome if you take big input by default (in this case you don't need input-big, input-big__wrapp and input-big__item classes). Look 2 point on [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

And try to describe the content of the tag on class naming. Form can't be block__input (it's not an input), use searchBar, searchForm or something like that